C_CPP/Strings and their functions/reverse_string.md

```diff
@@ -0,0 +1,28 @@
+<h1 text align="center"> Reversing words in a String</h1>
```

Comment:
please use [markdown](https://www.markdownguide.org/basic-syntax/)  formatting and not html